lib/index_documents.rb

```diff
@@ -0,0 +1,54 @@
+require_relative "../app"
+require 'gds_api/publishing_api_v2'
+
+class IndexDocuments
+  def process(message)
+    base_path = document_base_path(message)
+    topic_content_ids = topic_content_ids(message)
+
+    unless topic_content_ids.nil? # no topics field in links, don't update
+      topic_base_paths = topic_content_ids.map { |content_id| base_path(content_id) }
+      update_index(base_path, topic_base_paths)
+    end
```

Comment:
This was what I wanted to discuss with you: missing trello ticket information.

Should the ticket be updated and then add extra code changes to this PR?